src/components/SdkPage/Sdk/Sdk.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Link from '@docusaurus/Link'
+import '../../../css/custom.css';
+
+const Sdk = ({ title, icon, github, docs }) => {
+  return (
+    <div className="sdk-card">
+      <img className="sdk-image" src={`/${icon}`} alt={title} />
+      <p className="sdk-name">{title}</p>
+      <Link className="sdk-github-link" href={github} target='_github'>
+        GitHub
+      </Link>
+      <Link href={docs} target='_docs'>
```

Comment:
This `target` will open a new window, which is not what we want. Instead, we want it to navigate on the same page. I believe if you leave the target off, it should work?